src/cobalt_model/pagination_config.rs

```diff
@@ -125,13 +144,16 @@ impl PaginationConfigBuilder {
             permalink_suffix.unwrap_or_else(|| DEFAULT_PERMALINK_SUFFIX.to_owned());
         let order = order.unwrap_or(SortOrder::Desc);
         let sort_by = sort_by.unwrap_or_else(|| vec![DEFAULT_SORT.to_owned()]);
+        let mut date_index = date_index.unwrap_or_else(|| DEFAULT_DATE_INDEX.to_vec());
+        date_index.sort_unstable(); // keep the date indexes in logical order
```

Comment:
Sort or error?

Reply:
It's an intended sort, so if the user put month, day, year, we keep a logical order year, month, day to distribute the posts into.

Reply:
But what does it mean for the user to specify them out of order?  Are we just changing what they said, behind their backs?  Thats something I'm not normally a fan of.

Reply:
That's my issue: I can't think of a meaning to having them out of order. So I've done this to avoid friction but maybe an error is more appropriate, so there's no "magic"?

Reply:
We can always remove an error without impacting people.  Changing behavior might have an impact.